homeassistant-supervised/DEBIAN/postinst

```diff
@@ -127,3 +127,6 @@ systemctl start hassio-supervisor.service
 info "Installing the 'ha' cli"
 chmod a+x "${PREFIX}/bin/ha"
 
+info "Installation Complete!"
+info "Within a few minutes you will be able to reach Home Assistant at:"
+info "homeassistant.local:8123 or your machine's ip address:8123"
```

Comment:
```suggestion
info "Within a few minutes you will be able to reach Home Assistant at:"
info "http://homeassistant.local:8123 or using the IP address of your"
info "machine: http://IP_ADDRESS:8123"
```

Ideally, we should get the IP address for the primary interface and show it in the message.

But, that (+ this message) may be a bit of scope for this PR.

Reply:
Moved to #190